src/flow.ts

```diff
@@ -228,8 +221,24 @@ export class Flow {
    * @returns an array of Font objects corresponding to the provided `fontNames`.
    */
   static setMusicFont(...fontNames: string[]): Font[] {
+    // HACK during review to introduce the correct stacks step by step
+    switch (fontNames[0]) {
+      case 'Bravura':
+        CommonMetrics.fontFamily = 'Bravura,BravuraText';
```

Comment:
Yeah, we probably shouldn't use both Bravura and BravuraText, as they contain the same glyph code points.

Reply:
Roboto Slab already there. :)